testsuite/features/step_definitions/command_steps.rb

```diff
@@ -741,6 +741,10 @@
   @command_output, _code = $server.run("spacewalk-repo-sync -c #{arg1} -u #{arg2}", check_errors: false)
 end
 
+When(/^I call spacewalk\-repo\-sync for channel "(.*?)"$/) do |channel|
```

Comment:
This step description creates an ambiguity with the above step (`When(/^I call spacewalk\-repo\-sync for channel "(.*?)" with a custom url "(.*?)"$/) do |arg1, arg2|`), a simple change could solve this (e.g. `to sync the channel`)